{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "important"
		},
		"category": "csaf_vex",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "SUSE CVE-2018-6188",
				"title": "Title"
			},
			{
				"category": "description",
				"text": "django.contrib.auth.forms.AuthenticationForm in Django 2.0 before 2.0.2, and 1.11.8 and 1.11.9, allows remote attackers to obtain potentially sensitive information by leveraging data exposure from the confirm_login_allowed() method, as demonstrated by discovering whether a user account is inactive.",
				"title": "Description of the CVE"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "CVE-2018-6188",
				"url": "https://www.suse.com/security/cve/CVE-2018-6188"
			},
			{
				"category": "external",
				"summary": "SUSE Security Ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "external",
				"summary": "SUSE Bug 1077714 for CVE-2018-6188",
				"url": "https://bugzilla.suse.com/1077714"
			},
			{
				"category": "external",
				"summary": "Advisory link for openSUSE-SU-2023:0077-1",
				"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/OGS4NP24275NERRPQV6A6EONV6W3C2SK/"
			}
		],
		"title": "SUSE CVE CVE-2018-6188",
		"tracking": {
			"current_release_date": "2023-12-08T04:13:51Z",
			"generator": {
				"date": "2023-02-15T04:30:31Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf-vex.pl",
					"version": "1"
				}
			},
			"id": "CVE-2018-6188",
			"initial_release_date": "2023-02-15T04:30:31Z",
			"revision_history": [
				{
					"date": "2023-02-15T04:30:31Z",
					"number": "2",
					"summary": "Current version"
				},
				{
					"date": "2023-03-21T03:41:27Z",
					"number": "3",
					"summary": "Current version"
				},
				{
					"date": "2023-12-08T04:13:51Z",
					"number": "4",
					"summary": "Current version"
				}
			],
			"status": "interim",
			"version": "4"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "SUSE Enterprise Storage 3",
								"product": {
									"name": "SUSE Enterprise Storage 3",
									"product_id": "SUSE Enterprise Storage 3",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:ses:3"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE Enterprise Storage 4",
								"product": {
									"name": "SUSE Enterprise Storage 4",
									"product_id": "SUSE Enterprise Storage 4",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:ses:4"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE Enterprise Storage 5",
								"product": {
									"name": "SUSE Enterprise Storage 5",
									"product_id": "SUSE Enterprise Storage 5",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:ses:5"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud 6",
								"product": {
									"name": "SUSE OpenStack Cloud 6",
									"product_id": "SUSE OpenStack Cloud 6",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-openstack-cloud:6"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud 7",
								"product": {
									"name": "SUSE OpenStack Cloud 7",
									"product_id": "SUSE OpenStack Cloud 7",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-openstack-cloud:7"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE Package Hub 12",
								"product": {
									"name": "SUSE Package Hub 12",
									"product_id": "SUSE Package Hub 12",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:packagehub:12"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE Package Hub 12 SP1",
								"product": {
									"name": "SUSE Package Hub 12 SP1",
									"product_id": "SUSE Package Hub 12 SP1",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:packagehub:12:sp1"
									}
								}
							},
							{
								"category": "product_name",
								"name": "openSUSE Tumbleweed",
								"product": {
									"name": "openSUSE Tumbleweed",
									"product_id": "openSUSE Tumbleweed",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:tumbleweed"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python-Django",
								"product": {
									"name": "python-Django",
									"product_id": "python-Django",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python-Django-1.11.10-5.1",
								"product": {
									"name": "python-Django-1.11.10-5.1",
									"product_id": "python-Django-1.11.10-5.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:1.11.10:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python-Django-1.11.15-2.1",
								"product": {
									"name": "python-Django-1.11.15-2.1",
									"product_id": "python-Django-1.11.15-2.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:1.11.15:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python310-Django-4.2.11-2.1",
								"product": {
									"name": "python310-Django-4.2.11-2.1",
									"product_id": "python310-Django-4.2.11-2.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:4.2.11:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python311-Django-4.2.11-2.1",
								"product": {
									"name": "python311-Django-4.2.11-2.1",
									"product_id": "python311-Django-4.2.11-2.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:4.2.11:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python312-Django-4.2.11-2.1",
								"product": {
									"name": "python312-Django-4.2.11-2.1",
									"product_id": "python312-Django-4.2.11-2.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:4.2.11:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python36-Django-3.2.7-2.3",
								"product": {
									"name": "python36-Django-3.2.7-2.3",
									"product_id": "python36-Django-3.2.7-2.3"
								}
							},
							{
								"category": "product_version",
								"name": "python38-Django-3.2.7-2.3",
								"product": {
									"name": "python38-Django-3.2.7-2.3",
									"product_id": "python38-Django-3.2.7-2.3",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:3.2.7:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python39-Django-3.2.7-2.3",
								"product": {
									"name": "python39-Django-3.2.7-2.3",
									"product_id": "python39-Django-3.2.7-2.3",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:3.2.7:*:*:*:*:*:*:*"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django-1.11.10-5.1 as component of SUSE Package Hub 12",
					"product_id": "SUSE Package Hub 12:python-Django-1.11.10-5.1"
				},
				"product_reference": "python-Django-1.11.10-5.1",
				"relates_to_product_reference": "SUSE Package Hub 12"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django-1.11.15-2.1 as component of SUSE Package Hub 12 SP1",
					"product_id": "SUSE Package Hub 12 SP1:python-Django-1.11.15-2.1"
				},
				"product_reference": "python-Django-1.11.15-2.1",
				"relates_to_product_reference": "SUSE Package Hub 12 SP1"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Django-4.2.11-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Django-4.2.11-2.1"
				},
				"product_reference": "python310-Django-4.2.11-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Django-4.2.11-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Django-4.2.11-2.1"
				},
				"product_reference": "python311-Django-4.2.11-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Django-4.2.11-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Django-4.2.11-2.1"
				},
				"product_reference": "python312-Django-4.2.11-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python36-Django-3.2.7-2.3 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python36-Django-3.2.7-2.3"
				},
				"product_reference": "python36-Django-3.2.7-2.3",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python38-Django-3.2.7-2.3 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python38-Django-3.2.7-2.3"
				},
				"product_reference": "python38-Django-3.2.7-2.3",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python39-Django-3.2.7-2.3 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python39-Django-3.2.7-2.3"
				},
				"product_reference": "python39-Django-3.2.7-2.3",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of SUSE Enterprise Storage 3",
					"product_id": "SUSE Enterprise Storage 3:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "SUSE Enterprise Storage 3"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of SUSE Enterprise Storage 4",
					"product_id": "SUSE Enterprise Storage 4:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "SUSE Enterprise Storage 4"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of SUSE Enterprise Storage 5",
					"product_id": "SUSE Enterprise Storage 5:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "SUSE Enterprise Storage 5"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of SUSE OpenStack Cloud 6",
					"product_id": "SUSE OpenStack Cloud 6:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "SUSE OpenStack Cloud 6"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of SUSE OpenStack Cloud 7",
					"product_id": "SUSE OpenStack Cloud 7:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2018-6188",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2018-6188"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "django.contrib.auth.forms.AuthenticationForm in Django 2.0 before 2.0.2, and 1.11.8 and 1.11.9, allows remote attackers to obtain potentially sensitive information by leveraging data exposure from the confirm_login_allowed() method, as demonstrated by discovering whether a user account is inactive.",
					"title": "CVE description"
				}
			],
			"product_status": {},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2018-6188",
					"url": "https://www.suse.com/security/cve/CVE-2018-6188"
				},
				{
					"category": "external",
					"summary": "SUSE Security Ratings",
					"url": "https://www.suse.com/support/security/rating/"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1077714 for CVE-2018-6188",
					"url": "https://bugzilla.suse.com/1077714"
				},
				{
					"category": "external",
					"summary": "Advisory link for openSUSE-SU-2023:0077-1",
					"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/OGS4NP24275NERRPQV6A6EONV6W3C2SK/"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"SUSE Package Hub 12 SP1:python-Django-1.11.15-2.1",
						"SUSE Package Hub 12:python-Django-1.11.10-5.1",
						"openSUSE Tumbleweed:python310-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python311-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python312-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python36-Django-3.2.7-2.3",
						"openSUSE Tumbleweed:python38-Django-3.2.7-2.3",
						"openSUSE Tumbleweed:python39-Django-3.2.7-2.3"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
						"version": "3.0"
					},
					"products": [
						"SUSE Package Hub 12 SP1:python-Django-1.11.15-2.1",
						"SUSE Package Hub 12:python-Django-1.11.10-5.1",
						"openSUSE Tumbleweed:python310-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python311-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python312-Django-4.2.11-2.1",
						"openSUSE Tumbleweed:python36-Django-3.2.7-2.3",
						"openSUSE Tumbleweed:python38-Django-3.2.7-2.3",
						"openSUSE Tumbleweed:python39-Django-3.2.7-2.3"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2018-01-26T08:52:51Z",
					"details": "important"
				}
			],
			"title": "CVE-2018-6188"
		}
	]
}
